docs/source/supercheq/Supercheq_css.ipynb

```diff
@@ -5,28 +5,57 @@
    "id": "7769945b",
```

Comment:
<p>Can you put these all at the <code>h2</code> level? (i.e., <code>##</code> instead of <code>#</code>) This way the headings won't show up in the sidebar.    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/541/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>